tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -227,6 +227,8 @@ public:
    virtual RNTupleVersion GetFieldVersion() const { return RNTupleVersion(); }
    /// Indicates an evolution of the C++ type itself
    virtual RNTupleVersion GetTypeVersion() const { return RNTupleVersion(); }
+   /// View this field as an RFieldDescriptor
+   RFieldDescriptor AsFieldDescriptor() const;
```

Comment:
I like the idea but I wonder if it is acceptable to ignore the field <--> subfield relationship.  We should at least mention this in the comment, perhaps even in the method name.

Reply:
Tried to address this with the new `RDanglingFieldDescriptor` type.